src/com/projectkorra/rpg/worldevent/util/EventManager.java

```diff
@@ -101,7 +116,7 @@ public void startEvent(World world, WorldEvent event, boolean natural) {
 		}
 	}
 
-	public void endEvent(World world, WorldEvent event) {
+	public void endEvent(World world, WorldEvent event, boolean blacklisted) {
```

Comment:
I would recommend we add an overloaded method which calls this one with false to improve code readability and to not have to pass false in a bunch for a majority of the events.

```java
public void endEvent(World world, WorldEvent event) {
    endEvent(world, event, false);
}

Reply:
Thought about doing that after the fact but didn't implement it. Will change it